{"uuid":"bf2500f6-6f55-49de-8829-97964260e51b","historyId":"1d2d85b7b8b3d65ca24cbf3505ada17f","fullName":"org.example.NegativeAuthTests.givenNonExistentPostId_whenDeletePost_thenUnauthorized","labels":[{"name":"package","value":"org.example.NegativeAuthTests"},{"name":"testClass","value":"org.example.NegativeAuthTests"},{"name":"testMethod","value":"givenNonExistentPostId_whenDeletePost_thenUnauthorized"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"rest"},{"name":"subSuite","value":"org.example.NegativeAuthTests"},{"name":"host","value":"dunice-HP-Z240-Tower-Workstation"},{"name":"thread","value":"13376@dunice-HP-Z240-Tower-Workstation.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Удаление поста"},{"name":"epic","value":"Негативное тестирование"}],"links":[],"name":"givenNonExistentPostId_whenDeletePost_thenUnauthorized","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tStatus code 401 expected [401] but found [400]","trace":"java.lang.AssertionError: The following asserts failed:\n\tStatus code 401 expected [401] but found [400]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)\n\tat org.example.NegativeAuthTests.givenNonExistentPostId_whenDeletePost_thenUnauthorized(NegativeAuthTests.java:147)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\n\tat org.testng.TestRunner.run(TestRunner.java:596)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\n\tat org.testng.TestNG.run(TestNG.java:1031)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","description":"Проверка удаления поста с несуществующим идентификатором.","steps":[],"attachments":[],"parameters":[],"start":1725526283380,"stop":1725526283542}